### ISSUE TITLE
No multithreading or parallelism? Embarassing.

### ISSUE THREAD

**Issue by arunanshub** [EnterpriseQualityCoding/FizzBuzzEnterpriseEdition#533]
Hello devs,

We, as a faced-paced company, like to perform multiple tasks at the same time with no regards to the quality of the final product.

This begs the question that what are we supposed to do while this program executes? Pay our unpaid developers? Pfft! We cannot lose our capital.

Hence, we are here to talk about implementation of multithreading, concurrency, async IO, parallelism etc.

- This code should be able to launch multiple threads at a time so that we can reduce the amount of time wasted.
- Also, parallelism is no joke. Hence we want you to implement it in CUDA, for the most optimal performance possible.
- You can think of some other points by yourself, because we are fast-paced and don't have time to think.

And please make sure that you use the industry standards for coding. For example, variables should be only one character long, since reading long variables wastes valuable time and energy, given that we are a fast-paced enterprise.

Happy coding. 😊

**Comment by kevinkt1** [comment]
Agree. Just wanted to add on that mutex locks are **HIGHLY DISCOURAGED** due to the fast-paced environment that our software must operate within. I'd like to remind everyone of a saying that is often attributed to the highly successful Elon Musk: "Fail fast. Fail early. Fail often."

**Comment by CzajnikOfficial** [comment]
its not a bug its a feature

**Comment by kevinkt1** [comment]
> Agree. Just wanted to add on that mutex locks are **HIGHLY DISCOURAGED** due to the fast-paced environment that our software must operate within. I'd like to remind everyone of a saying that is often attributed to the highly successful Elon Musk: "Fail fast. Fail early. Fail often."

Let the threads compete. May social Darwinism select the fittest of the threads.

**Comment by arunanshub** [comment]
> Let the threads compete. May social Darwinism select the fittest of the threads.

We must secure the existence of our thread-pool and a future for async threads.
